### PR TITLE
Navigation for managing organizations

### PR DESCRIPTION
**Pull request by divbzero**
Navigation from managing "Your account" to managing an organization:
- Add "Your organizations" link to "Your account" sidebar
- Add organizations list to "Your organizations" page
- Add "Manage" button each organization in organizations list

Navigation for managing each organization:
- Add "Collaborators" management page for each organization
- Add sidebar for managing each organization

Together this glues together two pages needed for managing members of an organization (#11076, #11077, #11078, #11081) and accepting/declining an organization invite (#11083).

### CODE REVIEW

**Comment by divbzero** [comment]
Depends on #11208. Wait until that’s in before merging this PR.

File: warehouse/manage/views.py [hunk 1]
```diff
@@ -991,7 +1047,24 @@ def __init__(self, request):
 
     @property
     def default_response(self):
+        all_user_organizations = user_organizations(self.request)
         return {
+            "organizations": self.organization_service.get_organizations_by_user(
+                self.request.user.id
+            ),
+            **user_organizations(self.request),
```
Comment (ewdurbin) [inline warehouse/manage/views.py:1055]: I assume the query would be cached, but we don't even need to make the 3 calls to SQLAlchemy in the function if we

```suggestion
            **all_user_organizations,
```

File: warehouse/templates/manage/manage_base.html [hunk 2]
```diff
@@ -204,6 +204,14 @@ <h3 class="sidebar-section__title">{% trans %}Your account{% endtrans %}</h3>
                   {% trans %}Your projects{% endtrans %}
                 </a>
               </li>
+              {% if not request.flags.enabled(AdminFlagValue.DISABLE_ORGANIZATIONS) %}
```
Comment (ewdurbin) [inline warehouse/templates/manage/manage_base.html:207]: Just a smell on the double negative. I think It's OK as is... but 

`{% if request.flags.enabled(AdminFlagValue.ENABLE_ORGANIZATIONS) %}` is just screaming at me through this string.

File: warehouse/manage/views.py [hunk 3]
```diff
@@ -1077,6 +1150,23 @@ def create_organization(self):
         return self.default_response
 
 
+@view_config(
+    route_name="manage.organization.roles",
+    context=Organization,
+    renderer="manage/organization/roles.html",
+    uses_session=True,
+    require_methods=False,
+    # permission="manage:organization",
```
Comment (ewdurbin) [inline warehouse/manage/views.py:1159]: I forget what status of permissions are currently.... I assume we need to address them more holistically at a later time.

Comment (ewdurbin) [reply warehouse/manage/views.py:1159]: Ah, I see these are handled in 11342

File: warehouse/templates/manage/manage_base.html (hunk 2 above)
Comment (ewdurbin) [reply warehouse/templates/manage/manage_base.html:207]: Not worth addressing at this time. Just a note.

**Pull request merged**